### ISSUE TITLE
orangehrm system is not accessible - mobile ios

### ISSUE THREAD

**Issue by CoSNikO** [orangehrm/orangehrm#1815]
### OrangeHRM version

5.6

### Environment details

- OrangeHRM source: Sourceforge
- Platform: CentOS Stream 9
- PHP version: 7.4.33
- Database and version: MySQL 8.0.36
- Web server: Apache 2.4.57
- Browser: Safari

### Describe the bug

I can access my server thru a browser from this address: https://192.168.X.XXX/ when i enter the same address in mobile app it displays "orangehrm system is not accessible" error.

**Comment by devishke-orange** [comment]
Hi @CoSNikO,

Can you access your system from a mobile browser?

**Comment by CoSNikO** [comment]
Yes! Its fully functional. i can access it from safari from the same device. But enter the same address to app the message appears. 

This one is from app:
![viber_image_2024-02-26_10-50-53-203](https://github.com/orangehrm/orangehrm/assets/30650198/051f86b7-e4f4-4e63-9085-f5ecb694e376)

And this from safari:
![viber_image_2024-02-26_10-58-13-422](https://github.com/orangehrm/orangehrm/assets/30650198/e68e36c9-05dd-4adb-b9d1-cb1b318b58de)

**Comment by CoSNikO** [comment]
is anything i can do to check what is wrong?
